src/app/_component/common/Navigation/LoginLink.tsx

```diff
@@ -0,0 +1,18 @@
+import Link from "next/link";
+import { MouseEvent, ReactNode } from "react";
+
+interface NavigationProps {
+  userId: string | undefined;
```

Comment:
```suggestion
userId?: string;
```
옵셔널 체이닝으로 간결하게 사용할 수 있을거같네요!

Reply:
오! 좋은지적 감사합니다!
빠르게 수정하겠습니다~